examples/python/quickstart_sql.py

```diff
@@ -0,0 +1,75 @@
+
+from pyspark.sql import Column, DataFrame, SparkSession, functions
+from pyspark.sql.functions import *
+from py4j.java_collections import MapConverter
+from delta.tables import *
+import shutil
+import threading
+
+tableName = "tbltestpython"
+
+# Enable SQL/DML commands and Metastore tables for the current spark session.
+# We need to set the following configs 
+
+spark = SparkSession.builder \
+    .appName("quickstart") \
+    .master("local[*]") \
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
+    .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
+    .getOrCreate()
+
+# Clear any previous runs
+try:
+    spark.sql("DROP TABLE IF EXISTS " + tableName)
+except:
+    pass
+
+# Create a table
+print("############# Creating a table ###############")
+spark.range(0, 5).write.format("delta").mode("overwrite").saveAsTable(tableName)
```

Comment:
If we are converting the scala example here to have the sql range, we should for this as well